dnf/cli/output.py

```diff
@@ -1087,7 +1087,7 @@ def _add_line(lines, data, a_wid, po, obsoletes=[]):
 %s
 %s
 """ % ('=' * self.term.columns,
-       self.fmtColumns(((_('Package'), -n_wid), (_('Arch'), -a_wid),
+       self.fmtColumns(((P_('Package', 'Package', 1), -n_wid), (_('Arch'), -a_wid),
```

Comment:
Second parameter is not used in this case. But "Packages" seems nicer. :-). I changed it.